### PR TITLE
Fix Jinja2 template rendering with autoescape enabled

### PR DESCRIPTION
**Pull request by sujay0412**
## Why?

We are using aws-deployment-framework for deployments. In our AWS accounts we are solving an Security Hub findings.
One of the finding found is referenced as CWE-20,79,80 - Cross-site scripting(XSS) with  high security labelled.

The issue is in the Jinja2 library, as used by `initial_commit.py` files. There was no `autoescape=True`.
I added the `autoescape=True` attribute to resolve this.
With these changes, the finding related with XSS is resolved in our AWS account.
Please consider this in next release.

Security-hub finding issue -CWE-20,79,80 - Cross-site scripting

> User-controllable input must be sanitized before it's included in output used to dynamically generate a web page.
> Unsanitized user input can introduce cross-side scripting (XSS) vulnerabilities that can lead to inadvertedly running malicious code in a trusted context.

## What?

Description of changes:

1. Add `autoescape=True` to resolve the XSS issue while rendering Jinja2 templates.

---

By submitting this pull request, I confirm that you can use, modify, copy, and
redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: src/lambda_codebase/initial_commit/initial_commit.py [hunk 1]
```diff
@@ -555,7 +555,7 @@ def create_adf_config_file(
 ) -> FileToCommit:
     template = HERE / input_file_name
     adf_config = (
-        jinja2.Template(template.read_text(), undefined=jinja2.StrictUndefined)
+        jinja2.Template(template.read_text(), undefined=jinja2.StrictUndefined,autoescape=True)
```
Comment (alFReD-NSH) [inline src/lambda_codebase/initial_commit/initial_commit.py:558]: I'm not the right person to review this technically, but style-wise there should be a space after the comma:

```suggestion
        jinja2.Template(template.read_text(), undefined=jinja2.StrictUndefined, autoescape=True)
```

**Comment by sujay0412** [comment]
please review

**Review by sbkok — CHANGES_REQUESTED** [review]
Can you also add this change to the other Jinja2 Template call?
It is used here: [src/lambda_codebase/initial_commit/bootstrap_repository/adf-bootstrap/deployment/lambda_codebase/initial_commit/initial_commit.py](https://github.com/awslabs/aws-deployment-framework/blob/ef662ece1227d62de105dda450c98f2d0d79b83e/src/lambda_codebase/initial_commit/bootstrap_repository/adf-bootstrap/deployment/lambda_codebase/initial_commit/initial_commit.py#L513)

**Comment by sujay0412** [comment]
Hi @sbkok  based on your comment i updated a file which you mentioned please review , Thanks

**Review by javydekoning — APPROVED** [review]

**Pull request merged**